boards/arm/cc3220sf_launchxl/cc3220sf_launchxl.yaml

```diff
@@ -5,6 +5,7 @@ arch: arm
 toolchain:
   - zephyr
   - gnuarmemb
+  - xtools
```

Comment:
@galak, I wonder, would cc3220sf ever need to build with xtensa toolchain?

Reply:
this is not xtensa, this is xtools

Reply:
ah, OK.  MacOS.